.travis.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This file does not use directives, so the document markers are useless here. So I'd rather avoid them. Thus, it makes the file less readable as many people don't know about them (and the Travis doc does not use them).

This is why I'm asking you why you use them. I know that they exist, but I don't see any benefit here